### PR TITLE
Added userlist feature/controller/views tests, controller, view and b…

### PR DESCRIPTION
**Pull request by memunyawiri**
…asic styling. All tests currently passing
Please review :)

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Comment by AlexSusanu** [comment]
: + shipit + :

File: spec/helpers/user_list_helper_spec.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+#
+# describe UserListHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe UserListHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (i-hardy) [inline spec/helpers/user_list_helper_spec.rb]: Could delete this as it's not doing anything

File: spec/views/user_list/index.html.erb_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "user_list/index.html.erb", type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (i-hardy) [inline spec/views/user_list/index.html.erb_spec.rb]: This is also not testing anything

**Review by ckiteou — APPROVED** [review]
:shipit:

File: config/routes.rb [hunk 3]
```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
 
+  get 'user_list/index'
```
Comment (paulmillen) [inline config/routes.rb]: is this required?

Comment (memunyawiri) [reply config/routes.rb]: It is, otherwise our page doesn't load

File: spec/helpers/user_list_helper_spec.rb (hunk 1 above)
Comment (memunyawiri) [reply spec/helpers/user_list_helper_spec.rb]: done

File: spec/views/user_list/index.html.erb_spec.rb (hunk 2 above)
Comment (memunyawiri) [reply spec/views/user_list/index.html.erb_spec.rb]: and done

File: config/routes.rb (hunk 3 above)
Comment (paulmillen) [reply config/routes.rb]: :shipit:

**Review by dbugsy — APPROVED** [review]
:shipit: :shipit:

**Pull request merged**